.github/workflows/build-and-deploy.yaml

```diff
@@ -0,0 +1,56 @@
+on:
+  push:
+    branches:
+      - master
+      - TECH-1850
+
+name: Build Docker image and deploy it to Kubernetes cluster
+
+jobs:
+  deploy:
+    name: Build-Deploy
+    runs-on: ubuntu-latest
+    environment: production
+    env:
+      AWS_REGION: us-east-1
+      CLUSTER_NAME: maker-prod
```

Comment:
Do you think it is a security risk to expose deployment region and cluster name in a public repo?